backend/services/deployments/tests/tests/test_deployment.py

```diff
@@ -421,16 +410,20 @@ def test_device_deployments_already_installed(self, mongo):
                     # NOTE: asking for a deployment while having it already
                     # installed is special in the sense that the status of
                     # deployment for this device will be marked as 'already-installed'
-                    nextdep = dc.get_next_deployment(
-                        dev.fake_token,
-                        artifact_name=artifact_name,
-                        device_type=dev.device_type,
-                    )
+                    nextdep = {}
+                    try:
+                        dc.get_next_deployment(
+                            dev.fake_token,
+                            artifact_name=artifact_name,
+                            device_type=dev.device_type,
+                        )
+                    except ValueError:
+                        nextdep = None
```

Comment:
I guess this is one of those instances where the generated client fails to comprehend an empty response. However, one small nitpick, nextdep is not assigned any other value if `get_next_deployment` would return a valid response, perhaps it should be like this?
```suggestion
                    nextdep = {}
                    try:
                        nextdep = dc.get_next_deployment(
                            dev.fake_token,
                            artifact_name=artifact_name,
                            device_type=dev.device_type,
                        )
                    except ValueError:
                        nextdep = None
```

Or like this?
```suggestion
                    with pytest.raises(ValueError):
                        dc.get_next_deployment(
                            dev.fake_token,
                            artifact_name=artifact_name,
                            device_type=dev.device_type,
                        )
                        # assert nextdep == None can be dropped in this case.
```

Reply:
let me see what I can do with that lovely piece. and I like the raises approach -- I took it. thank you.